### ISSUE TITLE
Support for Yuzu

### ISSUE THREAD

**Issue by LokeYourLord** [Ecks1337/RyuSAK#62]
Since EmuSAK had support for Yuzu, it would be nice if RyuSAK could also be getting support for Yuzu since RyujiNX and Yuzu are very similar in many ways. Not for the shaders of course, but the rest of the features like Firmware, Title Keys and Game Mods. Thanks :)

**Comment by LokeYourLord** [comment]
Oh I see that you seem to have some aversion to Yuzu and don't answer why you won't support it. I've thus decided to fork it and will add Yuzu support down the line myself. Thank you for RyuSAK Ecks1337, appreciate it regardless and I'll add Yuzu support :)

**Comment by awsms** [comment]
>I've thus decided to fork it and will add Yuzu support down the line myself.
will u?

**Comment by LokeYourC3PH** [comment]
> >I've thus decided to fork it and will add Yuzu support down the line myself.
> 
> will u?

Was about to, and almost finished it actually. But then Yuzu mongoloid devs (*cough cough* Fernando) went crazy arrogant when I wanted to report a bug, took shit out of context, didn't let me defend myself, and banned me from their GitHub, and I swore myself to never touch Yuzu ever again until Fernando dies.

TLDR: Yuzu support will be added when their dev Fernando is dead, before that, forget it. Use RyujiNX, by now it has almost all the same features and no arrogant turds as developers.

**Comment by awsms** [comment]
cheers
![1731197](https://github.com/Ecks1337/RyuSAK/assets/48278661/59cd1208-739e-4a25-ab82-0a672d721636)

**Comment by LokeYourC3PH** [comment]
> cheers ![1731197](https://user-images.githubusercontent.com/48278661/239583187-59cd1208-739e-4a25-ab82-0a672d721636.jpeg)

The fact you went out of your way to track this down is great hahaha. But yeah indeed, once that vomit inducing turd ain't no more, you're getting Yuzu support day 1 from me, but I don't see that happening any time soon, so might as well get used to RyujiNX. The devs are nice, not as competent as the Yuzu team, but they have good intentions at least.

**Comment by awsms** [comment]
what annoys me the most about yuzu is the fact "early-builds" are behind a paywall. but zelda runs better on yuzu for me :sob:
